sdk/communication/azure-communication-callingserver/src/test/resources/session-records/DownloadContentAsyncTests.downloadContent404Async[1].json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
nit: Please remove [1] from recording files when copying them to the session records folder

Reply:
Actually when I removed that, the pipeline complained that it couldn't find the files. I re-added the `[1]`